File: test/fixtures/test.accessor.m
alpha = 1
beta = 1
d = BetaDistribution[ alpha, beta ]

x = N[ Array[# &, 100, {-2, 2}] ]

y = N[ MomentGeneratingFunction[ d, x ] ]


Print[ "alpha:\n" ]
Print[ alpha ]
Print[ "beta:\n" ]
Print[ beta ]
Print[ "data:\n" ]
Print[ x ]
Print[ "expected:\n" ]
Print[ y ]

File: test/fixtures/test.accessor.jl
using Distributions
using JSON

alpha = 1
beta = 1
d = Beta( alpha,beta )

x = linspace( .5, -100, 100 )

dmgf(t) = mgf(d, t )
y = map( dmgf, x )
println( y )

data = Dict([
	("alpha", alpha),
	("beta", beta),
	("data", x),
	("expected", y)
])

outfile = open("./test/fixtures/accessor.json", "w")
JSON.json(data)

write( outfile, JSON.json(data) )
